src/main/java/ru/hogwarts/school/repository/FacultyRepository.java

```diff
@@ -7,4 +7,7 @@
 
 public interface FacultyRepository extends JpaRepository<Faculty, Long> {
     List<Faculty> findAllByColor(String color);
+    List<Faculty> findByNameOrColorIgnoreCase(String name, String color);
```

Comment:
```suggestion
    List<Faculty> findByNameIgnoreCaseOrColorIgnoreCase(String name, String color);
```
потому что сейчас игнорируется регистр только при поиске по цвету